src/main/java/nextstep/security/authorization/manager/AuthoritiesAuthorizationManager.java

```diff
@@ -2,11 +2,24 @@
 
 import nextstep.security.authentication.Authentication;
 import nextstep.security.authorization.AuthorizationDecision;
+import nextstep.security.authorization.GrantedAuthority;
+import nextstep.security.authorization.role.NullRoleHierarchy;
+import nextstep.security.authorization.role.RoleHierarchy;
 
 import java.util.Collection;
 
 public class AuthoritiesAuthorizationManager implements AuthorizationManager<Collection<String>> {
 
+    private final RoleHierarchy roleHierarchy;
+
+    public AuthoritiesAuthorizationManager() {
+        this.roleHierarchy = new NullRoleHierarchy();
```

Comment:
```suggestion
        this(new NullRoleHierarchy());
```

소소하지만.. 주 생성자를 정해 하나의 생성자에서만 초기화를 담당하면 유지보수성을 높일 수 있습니다!